python/packages/autogen-core/src/autogen_core/base/intervention.py

```diff
@@ -14,6 +14,27 @@
 class DropMessage: ...
 
 
+def warn_if_none(result: Any, handler_name: str) -> Any:
```

Comment:
IMO there's no reason to return the value, and by doing so you're stripping the type away from it since you're using `Any`. I'd say best off to just check and warn and return as normal like before

Reply:
Ah, I had not realized that. Thank you for pointing it out!! I updated the code to reflect that.